Coordinates/detail attraction/style

jml74262 opened the pull request:
CRUD de las entidades 
coordinates
detail_attraction
style
Hecho por: Laura y Josue

Josue74262 committed:
CRUDS
-Se hace CRUD de "detail_attraction"
-Se hace CRUD de "coordinates"
-Ya están testeados bbs

Josue74262 committed:
CRUDS
-Se hace CRUD de "detail_attraction"
-Se hace CRUD de "coordinates"
-Ya están testeados bbs

Josue74262 committed:
CRUD Style
-Se añade el crud completo de style

Josue74262 committed:
CRUD Style
-Se añade el crud completo de style

EfrainJC commented on app/routes/detail_attraction.py:
```diff
@@ -0,0 +1,166 @@
+            detail_list.append(detail_data)
+        
+        return jsonify(detail_list)
+
```
Revisa la tu response con la tabla en la DB

EfrainJC commented on app/routes/detail_attraction.py:
```diff
@@ -0,0 +1,166 @@
+        # Devolver un mensaje de error en caso de excepción
+        return jsonify({'error': 'Error al obtener el detalle de atracción: ' + str(e)}), 400
+
+
```
Añade el metodo de get by id

Josue74262 committed:
Merge branch 'coordinates/detail_attraction/style' of https://github.com/Josephxrex/salle-around-me-back-end into coordinates/detail_attraction/style

Josue74262 committed:
Implementación de JWT
-Se implementa autenticación en:
coordinates, detail_attraction, style
-Se crea método de obtener por id de detail_attraction
-Se desactiva el sort para json, haciendo que se respete el orden en las respuestas con jsonify
